Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -36,9 +36,6 @@ SettingsWindow::SettingsWindow(QWidget* parent) : QDialog(parent)
     // Actual Settings UI
     SetupSettingsWidget();
 
-    MakeUnfinishedWarning();
-
-    content_inner->addWidget(m_warning_group);
     content_inner->addWidget(m_settings_outer);
 
     content->addLayout(content_inner);
```

Comment:
There's no longer any need for `content_inner`, and it adds extra margins that shouldn't be there. The `m_settings_outer` widget can be added directly to `content`.